### DISCUSSION TITLE
Can't generate slice_files.egsphant

### DISCUSSION THREAD

**Discussion by Ahsong9527** [nrc-cnrc/EGSnrc#944] [Questions] [answered]
Hello researchers !
Question about the **ctcreate**,l follow the <getting-started>7.4 Convert CT DICOM data to egsphant step,and then l can't generate slice_files.egsphant file
![image](https://user-images.githubusercontent.com/116752446/208875525-28e7bfc3-821e-48ee-a25a-669cf3800b44.png)
Why？l look forward to your answers,thanks!!!

**Comment by rtownson** [comment, marked as answer]
Replace $OMEGA_HOME with the actual path on your system (something like `/home/yourUsername/EGSnrc/HEN_HOUSE/omega/progs/...etc...`)

**Reply by Ahsong9527** [reply]
Thank you very much,l solved the question,btw,this path is set very strictly >_<

**Comment by Ahsong9527** [comment]
![image](https://user-images.githubusercontent.com/116752446/209067156-d7d5b28a-5eec-4af6-8bc5-782ce6b62fb5.png)
another question,I don't know why the job has failed.The right picture is the Error journal

**Reply by rtownson** [reply]
The GUI incorrectly reports that the job has failed quite often, this is a bug. Check the .egslst and other output to see whether or not the job actually crashed, and you can ignore that popup warning. You should generally read through all of the output anyway, to check for warnings or errors.
